ParabolInc/parabol
# chore(teams-limit): remove teams limit notifications after upgrade or removing a team

Fixes #7746

**In this PR:**

* Added an ability to update notifications (mark them as clicked and close related snackbar)
* Unpublish teams limit notifications if team was upgraded or removing a team fixed the limit



**How to test:**

- Uncomment `const enum Threshold {`
- Reset an org
```
r.db('actionDevelopment').table('Organization')
  .getAll(r.args(['PUT ORG ID HERE']), {index: 'id'})

  .update({
    featureFlags: r.row('featureFlags').default([]).setInsert('teamsLimit'),
    tierLimitExceededAt: null,
    scheduledLockAt: null,
    lockedAt: null
});
```

- Complete a meeting to exceed teams limit 
- See snackbar appeared and new notifications dot on the bell icon in header
- Do not open notifications center, do not close a snackbar
- Upgrade a team (or remove a team to fix the limit)
- See that teams limit snackbar was closed
- See that notification marked as clicked
